src/app/[locale]/dataset-card.tsx

```diff
@@ -21,10 +24,10 @@ export default function DatasetCard({dataset, locale}: Props) {
         </h3>
         <div className="mt-2 flex">
           <p className="text-xs text-gray-500 mr-10">
-            Owner: {dataset.publisher.name}
+            {t('owner')}: {dataset.publisher.name}
```

Comment:
Can we change 'owner' to 'publisher' instead (in all files, not specifically this one)? In the data we currently only have the publishers.